main.cpp

```diff
@@ -407,9 +429,22 @@ bool SpaceShip::dock()
     }
     return false;
 }
-void SpaceShip::makeLoop(int planetNum )
+int SpaceShip::makeLoop(int planetNum, int loopCount )
 {
-    ++planetNum;
+    int loop = 0;
+    while (loop <= loopCount)
+    {
+        ++loop;
+        if ( loop == planetNum )
+        {
+            std::cout << "Planet "<<planetNum<< " Say bye!" << std::endl;
+            this->totalLoops += loop;

```

Comment:
add spaces around your `<<` so your code is more readable.

`this` hasn't been introduced yet.   Don't do more work than you are asked to do.
- [ ] revise